test/unit/audited_hacks_test.rb

```diff
@@ -25,4 +25,114 @@ def test_user
     assert_equal User.current, audit.user
   end
 
+  class LoggingTest < ActiveSupport::TestCase
+    disable_transactional_fixtures!
+
+    setup do
+      @provider = FactoryBot.create(:provider_account)
+      User.current = provider.provider_account.users.first!
+
+      @audit_class = Audited.audit_class
+      @audit = FactoryBot.build(:audit, provider_id: provider.id)
+
+      audit_class.delete_all
+    end
+
+    attr_reader :provider, :audit_class, :audit
+
+    test '#logging_to_stdout?' do
+      Features::LoggingConfig.config.stubs(audits_to_stdout: false)
+      refute audit_class.logging_to_stdout?
+      refute audit.logging_to_stdout?
+
+      Features::LoggingConfig.config.stubs(audits_to_stdout: true)
+      assert audit_class.logging_to_stdout?
+      assert audit.logging_to_stdout?
+    end
+
+    test 'logging to stdout disabled' do
+      audit_class.stubs(:logging_to_stdout? => false)
+      audit_class.any_instance.expects(:log_to_stdout).never # I know, expecting a method never to be invoked sucks :/
```

Comment:
mmmm, I dunno if what I am about to suggest is a good idea or not but there it goes:
maybe we could use the gem `minitest-logger` and actually do assertions about the logging itself 😄

Reply:
Let's study it. For a different PR, of course.